### ISSUE TITLE
[TSDoc> Intro]: In scan mode screen reader is not announcing the information of the 'TypeDoc' link which is present under the list.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/rushstack-websites#151]
**Test Environment:**
OS Version: 22H2 (OS Build 25370.1)
Browser: Edge Dev (Version 115.0.1866.1 (Official build) dev (64-bit))
URL: https://tsdoc.org/
Screen reader: Narrator

**Repro Steps:**
1. Open the above URL in edge dev browser and login with v-.
2.Turn on the narrator through Ctrl+ windows+ Enter key and press caps lock+ space bar key to turn on the scan mode.
3. Navigate to the 'Intro' link and activate it.
4. Navigate to the 'TypeDoc' link which is present under list i.e bullets' through up/down arrow key.
5. Observe the issue.

**Actual Result:**
In scan mode screen reader is not announcing the information of the 'TypeDoc' link which is present under the list..
Screen reader remains silent.

**Expected Result:**
In scan mode screen reader should announce the information of the 'TypeDoc' link which is present under the list.

**Observation:**
Same issue repro with NVDA and Issue does not repro on tab mode.

**Notes:**

1. Same issue repro for all the links which is present under list i.e on bullets point.
2. Same issue repro on 'Typesense' link under '**Search'** dailog

**User Impact:**
Screen reader user will get impacted as user will not get the information and will not be able to make track of the content and will not be able to navigate properly.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

https://github.com/microsoft/rushstack-websites/assets/104483005/060306ca-9f7c-42c0-a1c5-6fb9595a60cc

<img width="960" alt="In scan mode screen reader is not announcing links information which is present under intro" src="https://github.com/microsoft/rushstack-websites/assets/104483005/bf26c231-6702-496d-a149-5c18a0219e25">
<img width="960" alt="Notes~ Search ~ Type sense link is not accessible in scan mode" src="https://github.com/microsoft/rushstack-websites/assets/104483005/2678927b-0aa9-4c78-9253-a8390e5ea6eb">

**Comment by Raisul123** [comment]
Issue has been fixed , Hence closing the issue
PFA

<img width="954" alt="Image" src="https://github.com/user-attachments/assets/9c7fd7c3-7c8d-41f7-97e7-df8b8179d025" />
